tests/test_templates.py

```diff
@@ -0,0 +1,80 @@
+import os.path
+from unittest.mock import patch
+
+from twisted.trial import unittest
+
+from sydent.util.emailutils import sendEmail
+from tests.utils import make_sydent
+
+
+class TestTemplate(unittest.TestCase):
+    def setUp(self):
+        # Create a new sydent
+        config = {
+            "general": {
+                "templates.path": os.path.join(
+                    os.path.dirname(os.path.dirname(__file__)), "res"
+                ),
+            },
+        }
+        self.sydent = make_sydent(test_config=config)
+
+    def test_jinja_template(self):
```

Comment:
it would be good to test that our escaping and non-escaping works properly (since it is a security aspect as well)

Reply:
Just curious-line 53 of this file has a test for escaping, should I beef it up more? Will definitely add a test for non-escaping (by that I assume you mean testing that the non-escaped plaintext doesn't add escapes?)

Reply:
Yeah, I see you have had a crack at it, sorry for missing it.
Testing that things won't be needlessly escaped is good, yeah